packages/itwinui-css/backstop/tests/assets/theme.js

```diff
@@ -33,49 +39,74 @@ class ThemeButton extends HTMLElement {
         justify-items: end;
         overflow: hidden;
       }
-      button[aria-label="Settings"] {
+      button[aria-label="Page settings"] {
         pointer-events: auto;
-        padding: 0.5rem;
+        padding: var(--iui-size-xs);
         font: inherit;
         border: none;
         border-radius: 50%;
-        background: transparent;
+        background-color: hsl(var(--iui-color-foreground-1-hsl) / 0.1);
+        backdrop-filter: blur(5px);
         display: inline-grid;
         cursor: pointer;
+        outline-offset: -1px;
+        transition: background-color var(--iui-duration-1) ease;
         -webkit-tap-highlight-color: transparent;
       }
       @media (pointer: coarse) {
-        button[aria-label="Settings"] {
-          padding: 1rem;
+        button[aria-label="Page settings"] {
+          padding: var(--iui-size-m);
         }
       }
-      button[aria-label="Settings"]:hover {
-        background: hsl(0 0% 50% / 0.3);
+      button[aria-label="Page settings"]:hover,
+      button[aria-label="Page settings"]:focus-within {
+        background-color: hsl(var(--iui-color-foreground-1-hsl) / 0.2);
       }
       .popup {
         visibility: hidden; 
         pointer-events: none;
-        box-shadow: 0 1px 5px hsl(0 0% 0% / 0.25);
+        box-shadow: var(--iui-shadow-1);
         background-color: var(--iui-color-background-1);
-        padding: 0.5rem 0.25rem;
-        margin: 0.25rem;
+        padding: var(--iui-size-xs) var(--iui-size-s);
+        margin: var(--iui-size-2xs);
+        display: flex;
+        flex-direction: column;
+        gap: var(--iui-size-xs);
+        opacity: 0;
+        user-select: none;
```

Comment:
why disable user-select?

Reply:
I had this happen a couple of times while working on the theme PR, quickly opening, clicking, and closing the popup.
![Screen Shot 2022-10-18 at 5 07 45 PM](https://user-images.githubusercontent.com/849817/196545100-8eb7234f-f11a-49d3-93fd-6d1d88bcd65d.png)

Reply:
Disabling user-select is generally very bad practice. 🫤

Can you share a screen recording of how this happened? I can't seem to reproduce no matter how fast i open->click->dismiss.

Reply:
Here's a screen recording of 2 ways I've been accidentally doing it.
1. Double clicking on the label.  I know a single click works, sometimes I accidentally double click.
2. After clicking the button, while moving the cursor to the body of the page i click & sometimes it registers as a click & drag.  In the video I'm purposefully trying to make it happen, but it has happened to me on accident when naturally working.

https://user-images.githubusercontent.com/849817/196744584-cb0a50c1-281c-4d6a-9541-c7375a693a91.mov

Reply:
Ok, as long as we are not messing with `user-select` in all our components 👍